projects/ad713x_fmcz/src/ad713x_fmc.c

```diff
@@ -348,6 +361,10 @@ int main()
 
 #endif /* IIO_SUPPORT */
 
+	ret = ad713x_channel_sync(ad713x_dev_1);
+	if (ret != 0)
+		return -1;
```

Comment:
return ret, so we are consistent with the rest of the code

Reply:
Done.